deploy/custom-resources.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: db-credential-secret
+  namespace: test
+stringData:
+  type: "mysql"
+  host: "mysql.mysql.svc.cluster.local"
+  port: "3306"
+  database: "gingersnap"
+  username: "gingersnap_user"
+  password: "Password!42"
```

Comment:
is this a different setup? the demo on mysql uses:
`database: airports`
`password: password`

Reply:
Updated.